### PR TITLE
Add tinygrad icon

### LINKED ISSUE #10795: Request: Tinygrad
### Brand Name

Tinygrad

### Website

https://tinygrad.org/

### Popularity Metric

> 23k ✨

https://github.com/tinygrad/tinygrad

### Official Resources for Icon and Color

tinygrad

https://raw.githubusercontent.com/tinygrad/tinygrad/master/docs/logo_tiny_light.svg

### Additional Comments

_No response_

### PR DESCRIPTION
**Pull request by 21st-centuryman**
**Issue:** closes #10795

**Popularity metric:**
> 23k ✨

https://github.com/tinygrad/tinygrad

### Checklist

- [x] I updated the JSON data in `_data/simple-icons.json`
- [x] I optimized the icon with SVGO or SVGOMG
- [x] The SVG `viewbox` is `0 0 24 24`

### Description

tinygrad

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -15567,6 +15567,11 @@
             "hex": "1477D1",
             "source": "https://www.tinkercad.com"
         },
+        {
+            "title": "Tinygrad",
```
Comment (service-paradis) [inline _data/simple-icons.json:15571]:
```suggestion
            "title": "tinygrad",
```
They always refer themselves as `tinygrad` with a lowercase `t`.

File: icons/tinygrad.svg [hunk 2]
```diff
@@ -0,0 +1 @@
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Tinygrad</title><path d="M1.846 7.385V9.23H0v1.846h1.846v3.692h3.692v-1.846H3.692v-1.846h1.846V9.23H3.692V7.385zm5.539 0V9.23H9.23V7.385zm3.692 1.846v5.538h1.846v-3.692h1.846V9.23h-1.846zm3.692 1.846v3.692h1.846v-3.692zm3.693-1.846v3.692h3.692v1.846H24V9.231h-1.846v1.846h-1.846V9.23Zm3.692 5.538h-3.692v1.846h3.692zm-14.77-3.692v3.692h1.847v-3.692z"/></svg>
```
Comment (service-paradis) [inline icons/tinygrad.svg:1]:
```suggestion
<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>tinygrad</title><path d="M1.846 7.385V9.23H0v1.846h1.846v3.692h3.692v-1.846H3.692v-1.846h1.846V9.23H3.692V7.385zm5.539 0V9.23H9.23V7.385zm3.692 1.846v5.538h1.846v-3.692h1.846V9.23h-1.846zm3.692 1.846v3.692h1.846v-3.692zm3.693-1.846v3.692h3.692v1.846H24V9.231h-1.846v1.846h-1.846V9.23Zm3.692 5.538h-3.692v1.846h3.692zm-14.77-3.692v3.692h1.847v-3.692z"/></svg>
```

Same here about the title casing.

**Review by service-paradis — CHANGES_REQUESTED** [review]
The icon is looking great, thank you @21st-centuryman 🎉 

I left minor comments about the name. Once solved, I will merge this one!

**Comment by 21st-centuryman** [comment]
@service-paradis 
the changes have been submitted.

**Review by service-paradis — APPROVED** [review]

**Pull request merged**